### PR TITLE
Deploy harbor on EastUS Dev Cluster

### PR DESCRIPTION
**Pull request by kforeverisback**
# Type of PR

- [X] Documentation changes
- [X] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [X] I have updated the documentation accordingly.
- [X] My code follows the code style of this project.
- [X] I ran lint checks locally prior to submission.
- [X] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR

- Add Harbor endpoint to EastUS Dev cluster.
- Document how to deploy Harbor in dev clusters
- Individual sequence diagrams for different Harbor use cases

## Does this introduce a breaking change

- [ ] YES
- [X] NO

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/898

### CODE REVIEW

**Review by AAkindele — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -685,6 +685,132 @@ http https://ngsa-cosmos-$ASB_DOMAIN_SUFFIX/healthz
...
+  * Or you can check existing dns entries for same region
+* Create a public dns entry (e.g. `harbor-core-westus2-dev` for WestUS2)
+  in `dns-rg/cse.ms` and put the public ip for the regional cluster
+  * You can find the public IP in Application Gateway for the cluster
+    (e.g `rg-ngsa-asb-dev/apw-aks-{HASH}-{REGION}` )
+  * Or you can check existing dns entries for same region
+* Add the settings below for Harbor (similar to NGSA apps) in Application Gateway:
+  * Backend pool
+  * Backend settings
+    * [Optional] If using a health probe, use `/api/v2.0/health` Harbor endpoint
```
[the remark this answers is not in the document]
Comment (kforeverisback) [reply README.md:704]: Will remove the `[optional]` keyword

File: README.md [hunk 2]
```diff
@@ -685,6 +685,132 @@ http https://ngsa-cosmos-$ASB_DOMAIN_SUFFIX/healthz
...
+  az acr import --source docker.io/goharbor/redis-photon:v2.5.3 -n acraks3i2qzkkxofr7c
+  az acr import --source docker.io/goharbor/registry-photon:v2.5.3 -n acraks3i2qzkkxofr7c
+  az acr import --source docker.io/goharbor/trivy-adapter-photon:v2.5.3 -n acraks3i2qzkkxofr7c
+  ```
+
+After that, modify the required YAML files to prepare for deployment:
+
+* In [helm-values.yaml](./spikes/harbor/helm-values.yaml):
+  * Change the `repository:` and `tag:` value pairs (13 of them) to point to the proper ACR repo and tag
+  * Set the Harbor portal admin password (`harborAdminPassword:` )
```
[the remark this answers is not in the document]
Comment (kforeverisback) [reply README.md:747]: Not currently, no. We can change the helm chart to read the values from CSI secret, but that would require changing the Helm chart.

File: README.md [hunk 3]
```diff
@@ -685,6 +685,132 @@ http https://ngsa-cosmos-$ASB_DOMAIN_SUFFIX/healthz
 
 ```
 
+## Deploy Harbor (spike)
```
[the remark this answers is not in the document]
Comment (kforeverisback) [reply README.md]: Moved the `spike/harbor/README.md`

**Review by AAkindele — APPROVED** [review]
🚀

**Pull request merged**